services/web/client/source/class/qxapp/Application.js

```diff
@@ -166,6 +167,17 @@ qx.Class.define("qxapp.Application", {
           "transition: background-color 0s linear 5000s, color 0s linear 5000s"
         );
       }
+    },
+
+    __addGeneralCss: function() {
+      const stylesheet = qx.ui.style.Stylesheet.getInstance();
+      stylesheet.addRule(
+        ".no-user-select, .no-user-select > svg",
+
+        "-webkit-touch-callout: none;" +
+        "-webkit-user-select: none;" +
+        "user-select: none;"
+      );
```

Comment:
I would refrain from adding global css like that but rather create this in the NodeUi, potentially making sure to not add the css twice ...

Reply:
Ok... will look into that